Riot/Modules/Room/VoiceMessages/VoiceMessagePlaybackView.swift

```diff
@@ -128,7 +137,10 @@ class VoiceMessagePlaybackView: UIView, NibLoadable, Themable {
         self.backgroundColor = theme.colors.background
         playButton.backgroundColor = theme.colors.background
         playButton.tintColor = theme.colors.secondaryContent
-        backgroundView.backgroundColor = theme.colors.quinaryContent
+        
+        let backgroundViewColor = self.backgroundViewColor ?? theme.colors.quinaryContent
```

Comment:
Yes I understand your POV, in fact this the `VoiceMessagePlaybackView.backgroundView.backgroundColor` we are modifying not the `VoiceMessagePlaybackView.backgroundColor`. But it can be confused with `backgroundColor` property sure.